packages/ui/src/viewer-table/InteractionPanel/ExpandAllButton.tsx

```diff
@@ -0,0 +1,25 @@
+import { useEffect } from 'react';
+import Button from '../../common/Button';
+import { useThemeContext } from '../../theme/ThemeContext';
+
+interface ExpandAllButtonProps {
+	setIsCollapsed: (isCollapsed: boolean) => void;
+	expandOnLoad?: boolean; // This prop is optional and defaults to false
+}
+
+const ExpandAllButton = ({ setIsCollapsed, expandOnLoad = false }: ExpandAllButtonProps) => {
+	const theme = useThemeContext();
+	const { Eye } = theme.icons;
+
+	useEffect(() => {
+		setIsCollapsed(expandOnLoad);
+	}, [expandOnLoad, setIsCollapsed]);
```

Comment:
Repeat comment about the button not controlling the state of other components on load.